src/py/flwr/cli/new/new.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2024 Flower Labs GmbH. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Flower command line interface `new` command."""
+
+import os
+from enum import Enum
+from string import Template
+from typing import Dict, Optional
+
+import typer
+from typing_extensions import Annotated
+
+from ..utils import prompt_options
+
+
+class MLFramework(str, Enum):
```

Comment:
The rest of the codebase follows this naming style:
```suggestion
class MlFramework(str, Enum):
```